app/src/pages/instance/pluginsPage/pluginsCatalog/pluginsCatalog.scss

```
@@ -0,0 +1,59 @@
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 24px;
+  color: var(--rp-ui-color-text-3);
+}
+
+.plugins-catalog-empty-description {
+  margin: 0 0 16px;
+  font-family: var(--rp-ui-base-font-family);
+  font-size: 13px;
+  line-height: 20px;
+  // --rp-ui-base-gray-47 never existed — it was the app's $COLOR--gray-47 spliced into kit
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)